linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -33,15 +32,26 @@ def reportStageStatus(stageName, stageStatus) {
         }
     }
 }
+def GetPerfServer(xmlName) {
+    switch(xmlName) {
+        case ~/TCP.*/:
+            return "net_perf"
+        case ~/UDP.*/:
+            return "net_perf"
+        case ~/FIO.*/:
+            return "stor_perf"
+    }
 
 pipeline {
   parameters {
     choice(choices: 'git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git\nhttps://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git', description: 'Kernel tree repo', name: 'KERNEL_GIT_URL')
     string(defaultValue: "master", description: 'Branch to be built', name: 'KERNEL_GIT_BRANCH')
     choice(choices: 'Ubuntu_16.04.3\nCentOS_7.4', description: 'Distro version.', name: 'DISTRO_VERSION')
     choice(choices: "kernel_pipeline_bvt.xml\nkernel_pipeline_fvt.xml\ntest_kernel_pipeline.xml", description: 'Which tests should LISA run', name: 'LISA_TEST_XML')
+    choice(defaultValue: "TCP_SRIOV_v4.xml" choices: "TCP_SRIOV_v4.xml\nUDP_SRIOV_v4.xml\nFIO_RAID0_Pipeline.xml", description: 'Which xml should LISA run for performance test', name: 'LISA_PERF_TEST_XML')
+    string(defaultValue: "", description: 'Options for perf test.', name: 'LISA_PERF_OPTIONS')
```

Comment:
what is LISA_PERF_OPTIONS, what is it supposed to look like? maybe give an example

Reply:
After talking with @chvalean , I will remove LISA_PERF_OPTIONS as we are doing standard runs with no modifications to performance variables .